package/Shaders/Lighting.hlsl

```diff
@@ -1521,7 +1521,7 @@ PS_OUTPUT main(PS_INPUT input, bool frontFace
 	float minWetnessAngle = 0;
 	minWetnessAngle = saturate(max(minWetnessValue, worldSpaceNormal.z));
 #		if defined(SKYLIGHTING)
-	float wetnessOcclusion = saturate(shUnproject(skylightingSH, float3(0, 0, 1)) * 10);
+	float wetnessOcclusion = pow(saturate(shUnproject(skylightingSH, float3(0, 0, 1))), 2);
```

Comment:
This likely pushes wetness into objects. Are we sure this is the best option?

Reply:
Idk how u think that. SH is now scaled 4pi times, which is just a bit greater than original 10. And squaring makes pushes wetness OUT, not in.

Reply:
Ah true, that's fine then.